### PR TITLE
Add follow options

### LINKED ISSUE #3: Option follow moved window if current workspace becomes empty
https://faq.i3wm.org/question/5049/move-goto-workspace-when-moving-a-container-empties-the-focused-workspace/

### PR DESCRIPTION
**Pull request by clinta**
-f follows a moved window to the destination workspace. -F follows a moved window to the destination workspace only if the current workspace is empty after the move.

Closes #3

### CODE REVIEW

File: quickswitch.py [hunk 1]
```diff
@@ -41,6 +41,8 @@
 workspace_number_re = re.compile("^(?P<number>\d+)(?P<name>.*)")
 default_dmenu_command = "dmenu -b -i -l 20"
 window_class_ignore_list = []
+follow = False
+followifempty = False
```
Comment (OliverUv) [inline quickswitch.py]: `follow_if_empty` please

File: quickswitch.py [hunk 2]
```diff
@@ -429,6 +448,11 @@ def main():
 
     args = parser.parse_args()
 
+    global follow
+    follow = args.follow
+    global followifempty
+    followifempty = args.followifempty
```
Comment (OliverUv) [inline quickswitch.py]: don't like the globals very much, but looking at the code doing this in a more functional way looks like it'll get pretty messy

Comment (clinta) [inline quickswitch.py]: I had the same misgivings. There's a lot of work to refactor to try to add those as arguments to `move_container_to_workspace` though.

**Comment by OliverUv** [comment]
Looks good and seems to work! I'll tag this as a new version in a week if I haven't run into any issues, though I've already tested all of my own usual use cases. I'm guessing AUR will pick up the change if I bump & tag the minor version, right?

**Comment by OliverUv** [comment]
Also, thanks! :100:

**Comment by clinta** [comment]
Yeah, if you tag a release I can mark the aur package out of date and the package maintainer should bump the version in the aur.

**Comment by OliverUv** [comment]
I have now tagged the latest as 2.7.0 @clinta

**Pull request merged**